patcher.sh

```diff
@@ -388,6 +396,7 @@ main ()
 			echo -e "-m, --hqmusic\t\t\tIf provided, high quality music will be used, otherwise low quality music will be used instead."
 			echo -e "-w, --systemwide\t\tIf provided, Linux will get installed systemwide, otherwise Linux will get installed portably. Has no effect on Android."
 			echo -e "-a, --appimage\t\t\tIf provided, an AppImage will get generated, otherwise the raw binary will get generated instead. Has no effect on Android."
+			echo -e "--patchopenssl\t\t\tIf provided, the raw binary will have OpenSSL 1.0.0 patched to point to libcurl. Has no effect if the -a flag is used."
```

Comment:
A shorthand flag would be nice for this. `-l` (patchopenssL) or `-p` (Patchopenssl) perhaps?

Reply:
I believe `-p` is already taken, but I went ahead and added `-l` as shorthand